src/lib/Fullscreen.js

```diff
@@ -91,27 +91,39 @@ class Fullscreen extends EventEmitter {
      * Fires events when the fullscreen state changes
      *
      * @private
-     * @param {HTMLElement|Event} [el] - Fullscreen element
+     * @param {HTMLElement|Event} el - Fullscreen element
      * @return {void}
      */
     fullscreenchangeHandler = (el) => {
         let enter = false;
 
-        if (this.isSupported()) {
-            if (this.isFullscreen()) {
-                enter = true;
-            }
-        } else if (!this.isFullscreen(el)) {
-            enter = true;
-        }
+        enter = (this.isSupported() && this.isFullscreen()) || (!this.isSupported() && !this.isFullscreen(el));
 
         if (enter) {
+            this.focusFullscreenElement(el);
             this.emit('enter');
         } else {
             this.emit('exit');
         }
     };
 
+    /**
+     * Focuses the element
+     *
+     * @private
+     * @param {HTMLElement|Event} el - Fullscreen element or event
+     * @return {void}
+     */
+    focusFullscreenElement = (el) => {
```

Comment:
This seems like it is pretty generic, maybe add to util.js?

Reply:
It is and it isn't, taking either a HTMLElement or Event and extracting out the HTMLElement to focus on seems a bit specific to this logic, maybe if this situation comes up again in another file we can move it then